app/models/User.coffee

```diff
@@ -26,11 +26,30 @@ module.exports = class User extends CocoModel
   @schema: require 'schemas/models/user'
   urlRoot: '/db/user'
   notyErrors: false
+  PERMISSIONS: {
+    COCO_ADMIN: 'admin',
+    SCHOOL_ADMINISTRATOR: 'schoolAdministrator',
+    ARTISAN: 'artisan',
+    GOD_MODE: 'godmode',
+    LICENSOR: 'licensor'
+  }
+
+  isAdmin: ->
+    p = @get('permissions')
+    return p and @PERMISSIONS.COCO_ADMIN in p
+  isLicensor: ->
+    p = @get('permissions')
+    return p and @PERMISSIONS.LICENSOR in p
+  isArtisan: ->
+    p = @get('permissions')
+    return p and @PERMISSIONS.ARTISAN in p
+  isInGodMode: ->
+    p = @get('permissions')
+    return p and @PERMISSIONS.GOD_MODE in p
+  isSchoolAdmin: ->
+    p = @get('permissions')
+    return p and @PERMISSIONS.SCHOOL_ADMINISTRATOR in p
```

Comment:
Do we like this more verbose version?  The permissions structure is nice, but less excited about the 3x LoC that don't seem to contribute too much here.

I think the overridden Backbone Model.get for User object means we'll always get an empty array if User.permissions isn't set, which is why the previous version worked.

`isAdmin: -> @PERMISSIONS.COCO_ADMIN in @get('permissions', true)`

Reply:
Ah yes, good call 👍